containers/unit_tests/TestOffsetView.hpp

```diff
@@ -336,8 +339,8 @@ void test_offsetview_unmanaged_construction() {
 
 template <typename Scalar, typename Device>
 void test_offsetview_unmanaged_construction_death() {
-  // Preallocated memory (Only need a valid address for this test)
-  Scalar s;
+  // Preallocated memory
+  Kokkos::View<Scalar, Device> s("s");
```

Comment:
I'd prefer if you did
```suggestion
  Kokkos::View<Scalar, Device> s("s");
  Scalar* ptr = s.data();  // obtain a pointer into the right address space
  // use ptr below
```